### DISCUSSION TITLE
Cache Error (Blue Screen) Pls help!

### DISCUSSION THREAD

**Discussion by tlaszlozoltan** [fedorovvl/tso_client#346] [General]
I have problem with use client. I use portable client.
I got always the cache error blue screen. "File loading failed. Please check your network connection, clean your browser cache or try again later."

![image](https://github.com/fedorovvl/tso_client/assets/65942080/72c0d5f3-3cb9-493b-84fc-1d6cbbe547c7)

I did step from this page
https://github.com/fedorovvl/tso_client/wiki/Troubleshooting

- I deleted ...\AppData\Local\tso_portable folder and launched client.exe sometimes, but that didn't help

- I downloaded and owerwrite client.swf, but that didn't help

- I put that 2 line into ...\etc\hosts file, but that didn't help
23.213.161.26 ubistatic-a.akamaihd.net 
23.213.161.12 ubistatic-a.akamaihd.net

- I see that xml file
https://ubistatic-a.akamaihd.net/0018/live/GFX_HASHED/loca/c38e118623925673f42dd04a0700d2250607e9ae.xml

- "If you see such a screen, the first reason could be a UbiCollect. Disable it and try accessing without it."
What does it mean? I tried find UbiCollect in pc but no results.

Does anyone have any other ideas?

**Comment by skelgaard** [comment]
download latest from https://github.com/fedorovvl/tso_client/releases
make sure you don't have any shortcuts with any parameters on it
cause some of the parameters can also make this happen.

try putting the exefile in a empty folder.
if that doesn't work, come to the discord and we can try and help you there...
